cli/script/command-executor.ts

```diff
@@ -970,6 +977,17 @@ function printAccessKeys(format: string, keys: AccessKey[]): void {
     }
 }
 
+function printSessions(format: string, sessions: AccessKey[]): void {
+    if (format === "json") {
+        printJson(sessions);
+    } else if (format === "table") {
+        printTable(["Machine", "Started"], (dataSource: any[]): void => {
```

Comment:
How about "Logged in" instead of "Started"? GitHub uses the phrase "Signed in" for this in their sessions list, and it sounds more natural than "Started", since the action that the user took to create the session was to actually login.